diff --git a/node_modules/update-browserslist-db/index.js b/node_modules/update-browserslist-db/index.js
index 1246914..e1f55c5 100644
--- a/node_modules/update-browserslist-db/index.js
+++ b/node_modules/update-browserslist-db/index.js
@@ -42,11 +42,12 @@ function detectLockfile() {
   let lockfileShrinkwrap = join(packageDir, 'npm-shrinkwrap.json')
   let lockfileYarn = join(packageDir, 'yarn.lock')
   let lockfilePnpm = join(packageDir, 'pnpm-lock.yaml')
-  let lockfileBun = join(packageDir, 'bun.lockb')
+  let lockfileBun = join(packageDir, 'bun.lock')
+  let lockfileBunBinary = join(packageDir, 'bun.lockb')
 
   if (existsSync(lockfilePnpm)) {
     return { file: lockfilePnpm, mode: 'pnpm' }
-  } else if (existsSync(lockfileBun)) {
+  } else if (existsSync(lockfileBun) || existsSync(lockfileBunBinary)) {
     return { file: lockfileBun, mode: 'bun' }
   } else if (existsSync(lockfileNpm)) {
     return { file: lockfileNpm, mode: 'npm' }
@@ -296,7 +297,7 @@ module.exports = function updateDB(print = defaultPrint) {
   } else if (lock.mode === 'pnpm') {
     updateWith(print, 'pnpm up caniuse-lite')
   } else if (lock.mode === 'bun') {
-    updateWith(print, 'bun update caniuse-lite')
+    updateWith(print, 'npm update caniuse-lite')
   } else {
     updatePackageManually(print, lock, latest)
   }
@@ -322,11 +323,8 @@ module.exports = function updateDB(print = defaultPrint) {
       )
     } else {
       print(
-        pico.red(
-          '\n' +
-            listError.stack +
-            '\n\n' +
-            'Problem with browser list retrieval.\n' +
+        pico.gray(
+          'Problem with browser list retrieval.\n' +
             'Target browser changes won’t be shown.\n'
         )
       )
